src/journal/journald-manager.h

```diff
@@ -209,6 +217,21 @@ void manager_dispatch_message(Manager *m, struct iovec *iovec, size_t n, size_t
 void manager_driver_message_internal(Manager *m, pid_t object_pid, const char *format, ...) _sentinel_;
 #define manager_driver_message(...) manager_driver_message_internal(__VA_ARGS__, NULL)
 
+#define JOURNAL_CONFIG_INIT                                                                     \
+        (JournalConfig) {                                                                       \
+                .forward_to_socket = (SocketAddress) { .sockaddr.sa.sa_family = AF_UNSPEC },    \
+                .storage = _STORAGE_INVALID,                                                    \
+                .forward_to_kmsg = false,                                                       \
+                .forward_to_syslog = false,                                                     \
+                .forward_to_console = false,                                                    \
+                .forward_to_wall = false,                                                       \
```

Comment:
you can drop these four lines. compound init defaults to zero anyway for unspecified fields.